packages/features/insights/hooks/useInsightsColumns.tsx

```diff
@@ -225,3 +252,33 @@ export const useInsightsColumns = ({
     ];
   }, [isHeadersSuccess, headers]);
 };
+
+function CopyButton({ label, value }: { label: string; value: string }) {
```

Comment:
I wonder if we should use the `useCopy` hook in here instead of just using navigator.clipboard...

Reply:
Yeah I agree

Reply:
great catch!